content/reference-strategy/index.md

```diff
@@ -34,27 +34,36 @@ the same name. To get the URI of the referenced resource you have to:
 1. Compose a record set URI: `https://country.register.gov.uk/records/`
 1. Concatenate the URI with the value: `https://country.register.gov.uk/records/GB`
 
-There is a special case here where a foreign key refers to a register register
-record. Then the meaning of the relation is “everything in the register
-described by the linked record”.
+There is a case that can be seen as a special case where a foreign key refers
+to a register register record. Then the meaning of the relation can be seen as
+“everything in the register described by the linked record”.
 
 1. Given a fieldname `register`
 1. And a value `country`
-1. Compose a register subdomain: `country.register.gov.uk`
-1. Compose a set URI: `https://country.register.gov.uk/records/`
+1. Compose a register subdomain: `register.register.gov.uk`
+1. Compose a record set URI: `https://register.register.gov.uk/records/`
+1. Concatenate the URI with the value: `https://register.register.gov.uk/records/country`
+1. Derive the record set URI: `https://country.register.gov.uk/records/`
+
+The last step is not specified anywhere so it is an internal convention more
+than anything a regular user might know. It is defined here in order to mirror
+the capabilities for CURIEs.
 
 #### Benefits
 
 * Potentially familiar to people familiar with the Relational Data Model.
 * Hints a single type of thing for any value in the field.
 
-#### Problems
+#### Limitations
 
-* To know a field contains a foreign key, you need to look up the register field
-in the field definition. If it is informed, the datatype is not “string” but
-“key”.
+* To know a field contains a foreign key, you need to look up the `register`
+  field in the field register. If it is informed, the original field is a “key”
+  of type defined in the `datatype` field.
 * Only able to reference one register per field.
-* Linking a full set of records is a special case.
+* You need a field for each register even if they have the same type of
+  relationship with the record (e.g. owned by, managed by).
```

Comment:
I don't understand this point.

Reply:
At all or part of it? Let me try again:

With foreign keys, the field has a local identifier so the field itself is dedicated to a single type of link (e.g. local-authority-eng). If you need to provide links to e.g. local-authority-nir as well, you need another column. Even though both have the same intention (relationship wise).

Reply:
Yes that makes sense now. I think it would be good to clarify that you "need a field for each register linked to"